src/modes/freehand.js

```diff
@@ -0,0 +1,75 @@
+const DrawPolygon = require('./draw_polygon');
```

Comment:
I would add a comment referring ppl to the module this is based off of, and describing the differences